packages/editor/types.ts

```diff
@@ -146,6 +146,7 @@ export interface EditorTheme extends Theme {
 	contentMaxWidth?: number;
 	marginLeft?: number;
 	marginRight?: number;
+	listTabSize?: string;
```

Comment:
Since this is also used for code blocks in the note editor, it may make sense to name this `tabSize`.
```suggestion
	tabSize?: string;
```

In the note viewer, the code block tab size also seems to be roughly the same as the list tab size:
![screenshot: Code block tab size is roughly the same as the list tab size](https://github.com/user-attachments/assets/21405bbc-762e-4c70-a270-f1c43fedc0de)

Reply:
As commented above, it may make more sense to only apply `listTabSize` to lists in the Markdown editor. As such, I'm marking this comment as resolved.